src/main/kotlin/com/jfrog/conan/clionplugin/toolWindow/ConanWindowFactory.kt

```diff
@@ -189,47 +190,52 @@ class ConanWindowFactory : ToolWindowFactory {
 
-                            addButton.addActionListener {
-                                conanService.runUseFlow(name, comboBox.selectedItem as String)
-                                val isRequired = conanService.getRequirements().any { it.startsWith("$name/") }
-                                addButton.isVisible = !isRequired
-                                removeButton.isVisible = isRequired
-                                Messages.showMessageDialog("${comboBox.selectedItem} added", "Library added to project", Messages.getInformationIcon())
-                            }
+                                add(addButton)
+                                add(removeButton)
+
+                                addButton.addActionListener {
+                                    conanService.runUseFlow(name, comboBox.selectedItem as String)
+                                    val isRequired = conanService.getRequirements().any { it.startsWith("$name/") }
+                                    addButton.isVisible = !isRequired
+                                    removeButton.isVisible = isRequired
+                                    Messages.showMessageDialog("${comboBox.selectedItem} added", "Library added to project", Messages.getInformationIcon())
+                                }
+
+                                removeButton.addActionListener {
+                                    conanService.runRemoveRequirementFlow(name, comboBox.selectedItem as String)
+                                    val isRequired = conanService.getRequirements().any { it.startsWith("$name/") }
+                                    addButton.isVisible = !isRequired
+                                    removeButton.isVisible = isRequired
+                                    Messages.showMessageDialog("${comboBox.selectedItem} removed", "Library removed from project", Messages.getInformationIcon())
+                                }
 
-                            removeButton.addActionListener {
-                                conanService.runRemoveRequirementFlow(name, comboBox.selectedItem as String)
                                 val isRequired = conanService.getRequirements().any { it.startsWith("$name/") }
                                 addButton.isVisible = !isRequired
                                 removeButton.isVisible = isRequired
-                                Messages.showMessageDialog("${comboBox.selectedItem} removed", "Library removed from project", Messages.getInformationIcon())
-                            }
+                            })
 
-                            val isRequired = conanService.getRequirements().any { it.startsWith("$name/") }
-                            addButton.isVisible = !isRequired
-                            removeButton.isVisible = isRequired
-                        })
+                            add(JCEFHtmlPanel("https://prev.conan.io/center/$name").component)
```

Comment:
This is the interesting line, the rest are style changes. The only weird thing is that it renders on top of the buttons, not after them, so we lose them right now. Can't easily find a way around it, maybe @czoido has better luck